storage/src/robocorp/storage/__init__.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from ._storage import get_assets_client
+
+
+__version__ = "0.1.0"
+version_info = [int(x) for x in __version__.split(".")]
+
+LOGGER = logging.getLogger(__name__)
+ASSETS_CLIENT = get_assets_client()
```

Comment:
I'm not sure whether `ASSETS_CLIENT` and `get_assets_client()` should actually be a part of the public API...

Reply:
Probably not. We also discussed earlier today that the client could be lazily initialized using session_cache/task_cache

Reply:
Yes and agree! Still, for me, `get_assets_client` is a "protected" API, since it's coming from a private file (*_storage.py*) but is publicly exposed to our own modules using it inside our own package.
And I think this way we bring a lot more clarity to the DX, because:
- *_storage.py* + `get_assets_client()` -> function publicly available for usage inside our own package
- *_storage.py* + `_get_assets_client()` -> function restricted for usage inside its own module only
- *storage.py* + `get_assets_client()` -> function publicly available for any kind usage, especially API interface

Now to make things even more clear, I'd import `get_assets_client as _get_assets_client` so we don't make it "public" accidentally during the package initializer. Will this kind of convention bring more value for you as well?